packages/collaborative-editing/src/module/settings-fields.ts

```diff
@@ -20,7 +20,7 @@ export const fields: DeepPartial<Field>[] = [
 		meta: {
 			interface: 'toggle',
 			width: 'half',
-			note: 'Hide the current user\'s avatar from collaborative editing indicators',
+			note: "Hide the current user's avatar from collaborative editing indicators",
```

Comment:
This is the result of manually running prettier. I noticed eslint and prettier don't automatically run, but I didn't make any changes there to keep this PR simple and easy to review. I could hook it up in a separate PR.